libs/designsystem/button/src/button.component.scss

```diff
@@ -10,14 +10,24 @@ $button-width: (
   md: 88px,
   lg: 220px,
 ) !default;
+$button-margin: utils.size('xxxs');
 
 @function button-width($key) {
   @return map.get($button-width, $key);
 }
 
 @mixin button-xs {
+  /*
+  * Extra small button needs a smaller click area to e.g. not overlap with other 
+  * interactive elements. The maximum target size we can safely do is the button's 
+  * minimum width plus its margin on each side
```

Comment:
Consider [grammar](https://www.hamilton.edu/academics/centers/writing/seven-sins-of-writing/4) 😜 
```suggestion
  * minimum width plus it's margin on each side
```

Reply:
According to your link its seems to be the right choice here:

> It’s is a contraction for “it is.” It’s is never a possessive.
> Its is the possessive for “it.”

Reply:
Damn, you're good!!! ![giphub](https://media2.giphy.com/media/jUwpNzg9IcyrK/giphy.gif)